public/apps/configuration/constants.tsx

```diff
@@ -106,17 +106,26 @@ export const CLUSTER_PERMISSIONS: string[] = [
   'cluster:admin/opendistro/reports/instance/list',
   'cluster:admin/opendistro/reports/instance/get',
   'cluster:admin/opendistro/reports/menu/download',
+  'cluster:admin/opensearch/ml/create_model_meta',
```

Comment:
There are other permissions like `cluster:admin/opensearch/ml/models/get`, is this new permission related to the `/ml/models/*` format, if so this new addition isn't consistent, does `cluster:admin/opensearch/ml/model/create_meta`

Reply:
This is not related to `/ml/models/*` format.  `/ml/models/*` is only for model document management (we saved all models in model index). These new actions like create model meta, upload/load/unload, are operations not just for model documents, they also have other code logic like checking, creating model instances, run model to generate results. So the name strategy is put all direct model document operations under `ml/models/*`, put all direct task documents operations under `ml/tasks/*`, for other operation, just list under `/ml/*`. When user configure permission, they have to know which action to add there, adding `ml/models/*` will have limited impact scope.

Reply:
I would say there are pros and cons. Put all these actions under `ml/models/*` can make it easier to configure all model related actions. But that also make it easier for user to configure more permissions

Reply:
FYI @shanilpa @jimishs I'm not sure if we are interested in project-level consistency for permissions name formats, here is an example where we saw a slightly different model from other permissions.  Consider creating an issue to follow up if you are interested in working this way in the future.